docs/installation/windows/basic-environment.md

```diff
@@ -8,6 +8,8 @@ MySQL:
  - [Install latest version](http://dev.mysql.com/downloads/mysql)
  - Log into MySQL terminal, it may look like `mysql -uroot -proot`
  - Create database: `create database jtalks character set utf8`
+ - Note that if you want to use full functionality of jcommune, you need to do a few simple steps described in `\jcommune\docs\installation\general-installation-guide.md`
```

Comment:
- NB: some functionality (like posting extended unicode symbols e.g. smiles) is not going to work with default MySQL configuration, read details in the [Advanced configuration of MySQL](link here)